### PR TITLE
Feature/intent to df transposed

### PR DESCRIPTION
**Pull request by MRyderOC**
PR regarding Issue #72.
Better documentation.
As discussed better memory usage for intent_proto_to_df in  https://github.com/GoogleCloudPlatform/dfcx-scrapi/blob/6bb8b70b5660e3d5d416500df8aa9ae5506833b0/src/dfcx_scrapi/core/intents.py#L155

### CODE REVIEW

File: src/dfcx_scrapi/core/intents.py [hunk 1]
```diff
@@ -68,11 +68,11 @@ def intent_proto_to_dataframe(
         """intents to dataframe
 
         Args:
-          obj, intent protobuf object
-          mode: (Optional) basic returns display name and training phrase as
-            plain text.
-          Advanced returns training phrase and parameters df broken out by
-            parts.
+          obj (Intent protobuf)
```
Comment (kmaphoenix) [inline src/dfcx_scrapi/core/intents.py:71]: @MRyderOC for documentation here, you can omit the types inside the docstring. The type is already provide in the function definition. This is how the remainder of the library is currently documented, so this provides consistency throughout the docs.

For example:
```
obj: Intent Protobuf
mode: The operational mode of the function which determines the output
  "basic" returns display name and training phrase as plain text.
  "advanced" returns training phrases broken out by parts with their parameters included.
```

File: src/dfcx_scrapi/core/intents.py [hunk 2]
```diff
@@ -583,22 +584,40 @@ def bulk_intent_to_df(
...
         """Extracts all Intents and Training Phrases into a Pandas DataFrame.
 
         Args:
-          agent_id, agent to pull list of intents
-          mode: (Optional) basic returns display name and training phrase as
-            plain text.
-          Advanced returns training phrase and parameters df broken out by
-            parts.
-          language_code: Language code of the intents being uploaded. Ref:
+          agent_id (str):
```
Comment (kmaphoenix) [inline src/dfcx_scrapi/core/intents.py:592]: Same here. Omit the typing inside the docstring itself in favor of the type hints in the function definition to keep it consistent/clean

**Review by kmaphoenix — CHANGES_REQUESTED** [review]
@MRyderOC couple of minor changes then we should be good to go.

**Comment by MRyderOC** [comment]
Changes have been made as we discussed.

**Review by kmaphoenix — APPROVED** [review]
LGTM

**Pull request merged**